src/lib/components/notification/notification.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
+	import { Toast } from 'flowbite-svelte';
+	import { slide } from 'svelte/transition';
+	import { CheckCircleSolid, CloseCircleSolid } from 'flowbite-svelte-icons';
+
+	export let message = 'Default message'; // Mensagem padrão
```

Comment:
O que acha de ao invés de deixar um comentário criar uma constante `DEFAULT_MESSAGE`?
```suggestion
        const DEFAULT_MESSAGE = 'Default Message'
        
	export let message = DEFAULT_MESSAGE
```

Reply:
resolvido commit - 70bebee7d5ec5ce7d7d32bdd9056b1de578f9bf5